production/ksonnet/loki/consul.libsonnet

```diff
@@ -0,0 +1,7 @@
+local consul = import "consul/consul.libsonnet";
+
+consul {
```

Comment:
I'm not sure if we should add this file if we don't make any changes.

Reply:
Sorry, I forgot to expose them to config.libsonnet